### DISCUSSION TITLE
Email notifications cannot be reconfigured

### DISCUSSION THREAD

**Discussion by room202** [community/community#127498] [Repositories]
### Select Topic Area

Bug

### Body

If I have registered and unsubscribed my email address in the “Email notifications” section of any repository,
I will not be able to register again.

Target Service
[Settings]-[Email notifications]

Procedure
1. register two e-mail addresses
2. delete the two registered e-mail addresses
3. register two e-mail addresses again
4. the email address can no longer be registered

**Comment by room202** [comment]
The problem was resolved before I knew it."

**Comment by jiriwetter** [comment]
Same problem. I am now unable to activate notifications. I enter a new email and when I press Activate and Update settings the page refreshes and nothing happens.
